web/react/stores/browser_store.jsx

```diff
@@ -12,43 +12,57 @@ function getPrefix() {
 // Also change model/utils.go ETAG_ROOT_VERSION
 var BROWSER_STORE_VERSION = '.5';
 
-module.exports = {
-    initialized: false,
+class BrowserStoreClass {
+    constructor() {
+        this.initialize = this.initialize.bind(this);
+        this.getItem = this.getItem.bind(this);
+        this.setItem = this.setItem.bind(this);
+        this.removeItem = this.removeItem.bind(this);
+        this.setGlobalItem = this.setGlobalItem.bind(this);
+        this.getGlobalItem = this.getGlobalItem.bind(this);
+        this.removeGlobalItem = this.removeGlobalItem.bind(this);
+        this.clear = this.clear.bind(this);
+        this.actionOnItemsWithPrefix = this.actionOnItemsWithPrefix.bind(this);
+        this.isLocalStorageSupported = this.isLocalStorageSupported.bind(this);
+
+        this.initialized = false;
```

Comment:
We were calling initialize lazily before so either we should remove the call to this.initialize() from the constructor or just remove this.initialized along with any other calls to this.initialize. As it is, this.initialized does nothing since it's initialized to false and then set to true almost immediately. Personally, I'm a fan of just doing it in the constructor since the initialization isn't really expensive.